pytrickle/client.py

```diff
@@ -130,31 +216,62 @@ async def _ingress_loop(self):
+                        frame.timestamp = max(0, frame.timestamp - self._first_video_timestamp)
+                    elif isinstance(frame, AudioFrame):
+                        if self._first_audio_timestamp is None:
+                            self._first_audio_timestamp = frame.timestamp
+                        # Rebase PTS ensuring non-negative
+                        frame.timestamp = max(0, frame.timestamp - self._first_audio_timestamp)
+                except Exception as e:
+                    logger.debug(f"Timestamp normalization skipped due to error: {e}")
+                
                 # Process frames directly in ingress loop
                 try:
                     if isinstance(frame, VideoFrame):
-                        logger.debug(f"Processing video frame with frame processor: {frame.tensor.shape}")
-                        
-                        # Direct async processing
-                        processed_frame = await self.frame_processor.process_video_async(frame)
-                        if processed_frame:
-                            output = VideoOutput(processed_frame, self.request_id)
-                            await self._send_output(output)
-                            logger.debug(f"Sent async processed video frame to egress")
+                        if getattr(self.frame_processor, "queue_mode", False):
+                            # Enqueue for background processing
+                            await self.frame_processor.enqueue_video_frame(frame)
                         else:
-                            logger.warning(f"Frame processor returned None for video frame")
+                            # Direct async processing
+                            processed_frame = await self.frame_processor.process_video_async(frame)
+                            if processed_frame:
+                                output = VideoOutput(processed_frame, self.request_id)
+                                await self._send_output(output)
+                                logger.debug(f"Sent async processed video frame to egress")
+                            else:
+                                logger.warning(f"Frame processor returned None for video frame")
                             
                     elif isinstance(frame, AudioFrame):
-                        logger.debug(f"Processing audio frame with frame processor: {frame.samples.shape}")
-                        
-                        # Direct async processing for audio
-                        processed_frames = await self.frame_processor.process_audio_async(frame)
-                        if processed_frames:
-                            output = AudioOutput(processed_frames, self.request_id)
-                            await self._send_output(output)
-                            logger.debug(f"Sent async processed audio frame to egress")
+                        if getattr(self.frame_processor, "queue_mode", False):
```

Comment:
```suggestion
                        if not self.frame_processor.queue_mode:
```

Same commen as above